lightning/src/ln/channel.rs

```diff
@@ -2832,13 +2857,35 @@ impl<Signer: Sign> Channel<Signer> {
 		}
 	}
 
+	/// Trigger the autoclose timer if it's in the starting position
+	pub fn maybe_trigger_autoclose_timer(&mut self, current_feerate: u32, new_feerate: u32, background_feerate: u32) -> bool {
+
+		// Verify the new feerate is at least superior by 30% of current feerate before to
+		// start a autoclose timer.
+		if new_feerate < current_feerate * 1300 / 1000 && new_feerate < background_feerate {
```

Comment:
Lets check this on each timer tick? ie if feerate goes up some, but then comes back down, we should reset the timer to 0.

Reply:
Yes, done in `check_autoclose`.